### DISCUSSION TITLE
AnsiConsole.Ask  and only ENTER as input

### DISCUSSION THREAD

**Discussion by DiagProf** [spectreconsole/spectre.console#561] [Q&A] [answered]
What is the right way so that AnsiConsole.Ask can deal with just ENTER?

When I try that

AnsiConsole.Ask<string>("Press Enter to navigate home");

 I have to enter a different character, otherwise ENTER is not accepted.

Another attempt would be that
 AnsiConsole.Ask("Press Enter to navigate home", "");
but now the brackets are displayed to me
![image](https://user-images.githubusercontent.com/56319388/135134812-bd08e3d1-d7a9-45c1-b414-1502c6d19e3f.png)

Does somebody has any idea? or is that with the empty () a bug?

**Comment by patriksvensson** [comment, marked as answer]
The `Ask` method is for text input. I would recommend using something like `System.Console.ReadKey(true)` instead.

**Reply by DiagProf** [reply]
Thank you!
